src/Views/Sidebar.vala

```diff
@@ -59,6 +47,10 @@ public class Tuba.Views.Sidebar : Box, AccountHolder {
 			}
 	};
 
+	static construct {
+		typeof (Widgets.EmojiLabel).ensure ();
+	}
```

Comment:
Nor why this suddenly became required

Reply:
Probably a `master` issue, not worth investigating yet